client/src/main/java/org/apache/rocketmq/client/consumer/DefaultMQPushConsumer.java

```diff
@@ -231,6 +234,7 @@ public DefaultMQPushConsumer(final String consumerGroup, RPCHook rpcHook, Alloca
         this.consumerGroup = consumerGroup;
         this.allocateMessageQueueStrategy = allocateMessageQueueStrategy;
         defaultMQPushConsumerImpl = new DefaultMQPushConsumerImpl(this, rpcHook);
+        this.pullIntervalPolicy = new BoundedExponentialPullIntervalPolicy(100, 300000);
     }
```

Comment:
300s is too long. 
For SUB latency-sensitive applications, long polling is suggested.
So fix pull interval policy is suggested, and the default pull interval should be zero.